addons/test_mail/tests/test_performance.py

```diff
@@ -399,7 +399,7 @@ def test_mail_composer_w_template(self):
         test_record = self.env['mail.test.ticket'].browse(self.test_record_full.id)
         test_template = self.env['mail.template'].browse(self.test_template_full.id)
 
-        with self.assertQueryCount(__system__=11, employee=12):
+        with self.assertQueryCount(__system__=14, employee=15):
```

Comment:
This shows that the exists added a bit everywhere has a big impact for no real reasons :) hence we think it is the caller's job to ensure template exists or correctly give value to mail methods.